SKILL.tsv

```diff
@@ -4812,7 +4812,7 @@ SKILL_20150717_004811	A character whose [Poison] is removed due to [Detoxify] wi
 SKILL_20150717_004812	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will last for 2 secs per Attribute level.
 SKILL_20150717_004813	The enemies that are in the range of [Scan] will fall into Confusion with a certain probability for 5 secs. The probability will increase by 10% per Attribute level.
 SKILL_20150717_004814	Enemies within rage of [Tear Gas] receives damage every 2 seconds. Damage increases per attribute level.
-SKILL_20150717_004815	Enemy falls under [Massive Bleeding] status when attack with [Back Step] while bleeding for 10 secs, the duration will increase by 1 per Attribute level. the Massive Bleeding will be decided based on character's STR, Physical ATK.
+SKILL_20150717_004815	Enemy falls under [Massive Bleeding] for 10 seconds when attacked with [Backstab] while bleeding. The duration will increase by 1 second per Attribute level. Massive Bleed damage is based on character's STR, Physical ATK.
```

Comment:
Change to: The enemy become afflicted with [Massive Bleeding] for 10 seconds when attacked with [Backstab] while affected by bleeding. Increases the duration by 1 second per attribute level. Massive Bleed damage is based on your STR and physical attack.